prisma/migrations/20230911183907_cooperations/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- AlterTable
+ALTER TABLE "student" ADD COLUMN     "cooperationID" INTEGER;
+
+-- CreateTable
+CREATE TABLE "cooperation" (
+    "id" SERIAL NOT NULL,
+    "tag" TEXT NOT NULL,
```

Comment:
Do we also need a Name that can be shown in the UI later on?

Reply:
True!